bot/exts/github/_issues.py

```diff
@@ -0,0 +1,107 @@
+from typing import Union
+
+import discord
+from aiohttp import ClientSession
+from bot.constants import Emojis
+from discord import Embed
+from discord.ext import commands
+from loguru import logger
+
+
+BAD_RESPONSE = {
+    404: "Issue/pull request not located! Please enter a valid number!",
+    403: "Rate limit has been hit! Please try again later!",
+}
+MAX_REQUESTS = 5
+REPO_CHANNEL_MAP = {
+    "793864456249278486": "reagurk",
+    "793864456249278487": "gurkbot",
+    "793864456249278488": "py-gurklang",
+    "793864456249278489": "branding",
+}
+
+
+class Issues:
+    """Cog that allows users to retrieve issues from GitHub."""
+
+    def __init__(self, http_session: ClientSession) -> None:
+        self.http_session = http_session
+
+    @staticmethod
+    def get_repo(channel: discord.TextChannel) -> str:
+        """Get repository for the particular channel."""
+        if str(channel) in REPO_CHANNEL_MAP.keys():
+            return REPO_CHANNEL_MAP[str(channel)]
+        else:
+            return ""
```

Comment:
```suggestion
            return None
```
oh wee, here I go explicit again.